alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+RUN apk --no-cache add curl python
```

Comment:
the alpine gcloud image is as minimal as possible and the intent is to 'add on' the extra stuff and not front load it with everything (eg. the current google/cloud-sdk-docker  image grew to 1+GB.

in the case for cloud source repos w/ git, the extension to support that maybe something like:

```dockerfile
FROM google/cloud-sdk
RUN apk --update add git
RUN gcloud components install beta
```